docs/streams/developer-guide/datatypes.html

```diff
@@ -34,7 +34,7 @@
   <div class="section" id="data-types-and-serialization">
     <span id="streams-developer-guide-serdes"></span><h1>Data Types and Serialization<a class="headerlink" href="#data-types-and-serialization" title="Permalink to this headline"></a></h1>
     <p>Every Kafka Streams application must provide SerDes (Serializer/Deserializer) for the data types of record keys and record values (e.g. <code class="docutils literal"><span class="pre">java.lang.String</span></code>) to materialize the data when necessary.  Operations that require such SerDes information include: <code class="docutils literal"><span class="pre">stream()</span></code>, <code class="docutils literal"><span class="pre">table()</span></code>, <code class="docutils literal"><span class="pre">to()</span></code>, <code class="docutils literal"><span class="pre">repartition()</span></code>, <code class="docutils literal"><span class="pre">groupByKey()</span></code>, <code class="docutils literal"><span class="pre">groupBy()</span></code>.</p>
-    <p>You can provide SerDes by using either of these methods:</p>
+    <p>You can provide SerDes by using either of these methods, but you must use at least one of these methods:</p>
     <ul class="simple">
       <li>By setting default SerDes in the <code class="docutils literal"><span class="pre">java.util.Properties</span></code> config instance.</li>
       <li>By specifying explicit SerDes when calling the appropriate API methods, thus overriding the defaults.</li>
```

Comment:
Think you can do a quick find&replace to standardize all of these `SerDes` to just `Serdes`, since that's how we now spell it everywhere else?

Reply:
Sure, just did it for the `datatypes` page. Do you think we should do it for all uses in `dsl-api`?

Reply:
Hm, dare I ask how many pages do this? Actually you should be able to do it using `sed` to programmatically go through each of the pages (if you've never used `sed` before it's a good thing to learn, you should be able to figure out how it works from googling for find&replace examples)

That said, let's leave it out of this PR. Can you just file a ticket to do this in a followup PR? Then I think this one can be merged